pkg/server/controller.go

```diff
@@ -268,20 +269,24 @@ func (ctrl *Controller) serverMux() (http.Handler, error) {
 			routes = append(routes, []route{
 				{"/render", h},
 				{"/render-diff", h},
-				{"/merge", h},
 				{"/labels", h},
 				{"/label-values", h},
 				{"/export", h},
+				{"/merge", h},
+				{"/api/exemplars:merge", h},
+				{"/api/exemplars:query", h},
 			}...)
 		}
 	} else {
 		routes = append(routes, []route{
 			{"/render", ctrl.renderHandler()},
 			{"/render-diff", ctrl.renderDiffHandler()},
-			{"/merge", ctrl.mergeHandler()},
 			{"/labels", ctrl.labelsHandler()},
 			{"/label-values", ctrl.labelValuesHandler()},
 			{"/export", ctrl.exportHandler()},
+			{"/merge", ctrl.exemplarsHandler().MergeExemplars},
+			{"/api/exemplars:merge", ctrl.exemplarsHandler().MergeExemplars},
+			{"/api/exemplars:query", ctrl.exemplarsHandler().QueryExemplars},
```

Comment:
@kolesnikovae Forgot to ask you about this today, but why use colons in URL paths? I feel like it's pretty uncommon. I'd rather do something like:
```
			{"/api/exemplars/merge", ctrl.exemplarsHandler().MergeExemplars},
			{"/api/exemplars", ctrl.exemplarsHandler().QueryExemplars},
```

Reply:
This is a pretty popular approach to denote custom action/method: https://cloud.google.com/apis/design/custom_methods. `/` is fine but I want to make `/api/exemplars` a collection of exemplars. `/api/exemplars/merge` would contradict to the planned endpoints:

**GET** `/api/exemplars` – Lists exemplars' properties (e.g profile_id).
**GET** `/api/exemplars/{profile_id}` – Accesses specific exemplar (an associated profile, meta, timestamps, and labels)

Also, I was thinking about renaming `query` to `search`, to avoid confusion with the URL parameter.